src/content/dizi-onerilerim.md

```diff
@@ -168,35 +168,35 @@ Konusu ilginç ve devamının gelmeyeceğini, devam etse nereye kadar edebilece
 
 ### The Order
 
 Son zamanlarda sürekli izleyip durduğum ayin, satanizm ve kara büyüyle ilgili bu diziler yüzünden umarım başıma herhangi bir iş gelmez... Üstelik hepsini de beğeniyorum. Bu dizi de dahil. Üstelik oldukça güzel ve gerçekten iyi işlenmiş bir dizi olduğunu söylemem gerekiyor. Bir okulda yürütülen gizli bir satanist organizasyonunu konu alan bu dizi hem gizli örgütün anlamına, hem de _kurtadam_ terimine yepyeni bir anlam katıyor. İzlenebilecek güzel ve sürükleyici bir dizi olan The Order'ın fragmanına [buradan](https://www.youtube.com/watch?v=iD-cFgDDuqM) ulaşabilirsiniz.
 
 ### BoJack Horseman
 
-Karakterleri, havası, hikayesi, yönetmeni, seslendirmesi, oyunculuğu, inandırıcılığı, bir sonraki bölüm merakı, kısacası ne ararsanız bu dizi onu size dört dörtlük bir şekilde veriyor. Özellikle karakterler, hava ve oyunculuk konusunda daha önce izlediğim hiçbir (yetişkin) animasyonda görmediğim derecede kaliteli işlenmiş, olayları birbirine bağlamayı çok iyi becermiş, sezonlar boyunca sürükleyici bir hikaye geliştirmiş, sürekli değişen ortamlar ve yeni karakterlerle hikayeyi güçlendirmiş ve diziye renk katılmış. Dizinin 6 sezonu boyunca bir bölümünde bile geçmediğim yaklaşık 1 dakikalık jenerik videosu, tam yerinde girişiyle, olaylara göre değişen arka plan ve karakterleriyle beni kendine bağımlı eden bu jeneriğin ilk sezondaki hâlini [buradan](https://www.youtube.com/watch?v=rQvIR1oL1vE) izleyebilirsiniz. Daha fazla söylemek isterim ancak daha nasıl anlatacağımı bilmiyorum... İzleyin, sarıyor, fragmanını da [buradan](https://www.youtube.com/watch?v=i1eJMig5Ik4) bulabilirsiniz.
+Karakterleri, havası, hikâyesi, yönetmeni, seslendirmesi, oyunculuğu, inandırıcılığı, bir sonraki bölüm merakı, kısacası ne ararsanız bu dizi onu size dört dörtlük bir şekilde veriyor. Özellikle karakterler, hava ve oyunculuk konusunda daha önce izlediğim hiçbir (yetişkin) animasyonda görmediğim derecede kaliteli işlenmiş, olayları birbirine bağlamayı çok iyi becermiş, sezonlar boyunca sürükleyici bir hikâye geliştirmiş, sürekli değişen ortamlar ve yeni karakterlerle hikâeyi güçlendirmiş ve diziye renk katılmış. Dizinin 6 sezonu boyunca bir bölümünde bile geçmediğim yaklaşık 1 dakikalık jenerik videosu, tam yerinde girişiyle, olaylara göre değişen arka plan ve karakterleriyle beni kendine bağımlı eden bu jeneriğin ilk sezondaki hâlini [buradan](https://www.youtube.com/watch?v=rQvIR1oL1vE) izleyebilirsiniz. Daha fazla söylemek isterim ancak daha nasıl anlatacağımı bilmiyorum... İzleyin, sarıyor, fragmanını da [buradan](https://www.youtube.com/watch?v=i1eJMig5Ik4) bulabilirsiniz.
```

Comment:
Should this really be `hikâeyi` instead of `hikâyeyi`?

Reply:
oops.